idseq_dag/steps/run_srst2.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from idseq_dag.engine.pipeline_step import PipelineStep
+from idseq_dag.util.s3 import fetch_from_s3
+import idseq_dag.util.command as command
+
+class PipelineStepRunSRST2(PipelineStep):
+    '''
+    srst2 is used to detect resistance genes.
+    See: https://github.com/katholt/srst2
+    '''
+    def run(self):
+        ''' Invoking srst2 '''
+        db_file_path = fetch_from_s3(self.additional_files["resist_gene_db"], self.output_dir_local)
+        min_cov = str(self.additional_attributes['min_cov'])
+        n_threads = str(self.additional_attributes['n_threads'])
+        if len(self.input_files_local[0]) == 2:
+            fwd_rd = self.input_files_local[0][0]
+            rev_rd = self.input_files_local[0][1]
+            # Rename to format srst2 expects
+            command.execute('mv ' + fwd_rd + ' R1_001.fastq.gz')
+            command.execute('mv ' + rev_rd + ' R2_001.fastq.gz')
+            srst2_params = [
+                'srst2', '--input_pe', 'R1_001.fastq.gz', 'R2_001.fastq.gz', '--forward', '001', '--reverse', '001',
```

Comment:
how come `forward` and `reverse` arguments are the same? Shouldn't it be R1_001 and R2_001? Please try it and let me know. Thanks!

Reply:
yes, R1_001 and R2_001 work as well -- I think I might have misunderstood the srst2 documentation, thanks !